.circleci/workflows.yml

```diff
@@ -2898,6 +2898,14 @@ linux-x64-contributor-workflow: &linux-x64-contributor-workflow
     - build:
         requires:
           - node_modules_install
+
+    # In subsequent jobs, we use some contexts that are restricted to members of the Cypress organization.
+    # This job will allow for a Cypress member to approve and run the rest of the restricted jobs in the pipeline after the contributor code has been reviewed.
```

Comment:
How/where does a Cypress member approve and run the rest of the jobs? Should we document this somewhere like the [contributing doc](https://github.com/cypress-io/cypress/blob/develop/CONTRIBUTING.md)?

Reply:
In the Circle UI, next to the job there will be a thumbs up icon that you can click which brings up a modal to click "Approve". Circle has it documented [here](https://circleci.com/docs/workflows/#holding-a-workflow-for-a-manual-approval)... the screenshots seem a little outdated

Reply:
Added something to the contributing doc to mention this